cub/cub/iterator/arg_index_input_iterator.cuh

```diff
@@ -245,12 +241,6 @@ public:
     itr += offset;
     offset = 0;
   }
-
-  /// ostream operator
-  friend std::ostream& operator<<(std::ostream& os, const self_type& /*itr*/)
-  {
-    return os;
-  }
```

Comment:
Remark: This is technically a breaking change, but since we are going for the CCCL 3.0 release and the operator did not do anything useful, I am ok with dropping it.

Reply:
Btw, the usual workaround would be to:
```c++
#if !_CCCL_COMPILER(NVRTC)
  friend std::ostream& operator<<(std::ostream& os, const self_type& /*itr*/)
  {
    return os;
  }
#endif // !_CCCL_COMPILER(NVRTC)
```

Reply:
Just curious, if we were constrained to keep the operator, the way forward would be to change implementation of  `ArgIndexInputIterator` to derive from a base class suitable for NVRTC which would add the friended operator?

Reply:
I think using a preprocessor directive to disable the offending part for NVRTC is totally fine. We do this in a lot of places. But your solution would work as well.